src/swagger-specs/destination-authoring.yaml

```diff
@@ -0,0 +1,3286 @@
+      responses:
+        "200":
+          description: "A successful response returns HTTP status 200 with details of all destination configurations."
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/destinationResponse"
+        '401':
+          description: "Unauthorized. A mandatory header might be missing from the request or you may be missing permissions to access the resource. Verify your request and try again."              
+        '403':
+          description: "Forbidden. The requester is not authorized to access the resource or a mandatory header might be missing from the request. Verify your request and try again."
+#      security:
+#        - dest_auth:
+#            - "write:dest"
+#            - "read:dest"
+    post:
+      tags:
+        - "Destination configurations"
+      summary: "Create a destination configuration"
+      description: "You can create a new destination configuration by making a POST request to the `/destinations` endpoint."
+      operationId: "addDestination"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - in: "body"
+          name: "body"
+          description: "Destination configuration object for a new destination. Refer to the Destination SDK documentation for an [example of a destination configuration](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/functionality/destination-configuration.html?lang=en#example-configuration)."
+          required: true
+          schema:
+            $ref: "#/definitions/destination"
+      responses:
+        "201":
+          description: "A successful response returns HTTP status 201 with details of your newly created destination configuration."
+          schema:
+            $ref: "#/definitions/destinationResponse"
+        "400":
+          description: "Bad request. The request payload was incorrectly formatted. Check your payload formatting before trying again."
+        '401':
+          description: "Unauthorized. A mandatory header might be missing from the request or you may be missing permissions to access the resource. Verify your request before trying again."              
+        '403':
+          description: "Forbidden. The requester is not authorized to access the resource or a mandatory header might be missing from the request. Verify your request before trying again."
+#        "405":
+#          description: "Invalid input"
+#      security:
+#        - dest_auth:
+#            - "write:dest"
+#            - "read:dest"
+  /destinations/{INSTANCE_ID}:
+    get:
+      tags:
+        - "Destination configurations"
+      summary: "Retrieve a destination configuration by ID"
```

Comment:
Another new standardization for `summary` fields is that we're dropping the "by ID" clause and letting the `description` field take care of that specificity instead. So this would be simplified down to just "Retrieve a destination"